fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/sound/sound.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright (c) 2023 FlyByWire Simulations -->
-<!-- SPDX-License-Identifier: GPL-3.0 -->
```

Comment:
Maybe we still want to have this header? Also in the other files.

Reply:
I'll add them again. I did that when I initially did the move but this time I skipped that.